tests/integration-tests/conftest.py

```diff
@@ -204,13 +204,17 @@ def _cluster_factory(cluster_config):
 
 def _write_cluster_config_to_outdir(request, cluster_config):
     out_dir = request.config.getoption("output_dir")
+
+    # sanitize test name
+    test = request.node.nodeid.split("::", 1)
+    test_file = test[0]
+    test_name = test[1].replace("/", "-")
```

Comment:
shall we do a replace on the os.path.sep? Also I'm wondering if we should simply escape this or use a different separator since `-` is already used by pytest to separate test parameters values

Reply:
The "/" is not the os.path.sep in Windows, but I'm facing an error like:

```
FileNotFoundError: [Errno 2] No such file or directory: 'tests_outputs/1571825679.8591163.out/clusters_configs/dcv/test_dcv.py-test_dcv_configuration[eu-west-1-c5.xlarge-centos7-sge-5678-192.168.1.1/32-/myshared].config'
```

Reply:
I changed the separator to "_"